app/enterprise/2.1.x/kong-manager/authentication/ldap.md

```diff
@@ -59,3 +61,9 @@ Learn more about these properties in [Session Security in Kong Manager](/enterpr
 After starting Kong with the desired configuration, you can create new *Admins*
 whose usernames match those in the AD. Those users will then be able to accept
 invitations to join Kong Manager and log in with their LDAP credentials.
+
+### Using Service Directory Mapping on the CLI
```

Comment:
There are double headings for this, so you'll need to remove the heading from the include, or remove this heading here. 
![image](https://user-images.githubusercontent.com/3756245/93141548-00695e00-f6aa-11ea-80bc-f906b3230f6d.png)

Reply:
Thanks, Terry. This was still a WIP, which I should have noted on the PR. Appreciate the input.

Reply:
I removed the heading from the include, so this is resolved. See https://deploy-preview-2310--kongdocs.netlify.app/hub/kong-inc/ldap-auth/#using-service-directory-mapping-on-the-cli